.utility/push-docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e  # Exit on error
+
+echo "Welcome to loklak Docker hub push script"
+
+if [ "$TRAVIS_REPO_SLUG" != "loklak/loklak_server" ]; then
+    echo "Skipping Docker push for repo $TRAVIS_REPO_SLUG"
+    exit 0
+fi
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+    echo "Skipping Docker push for pull request"
```

Comment:
Shouldn't the message be "Skipping the Docker push for current build" rather than pull request as the variable is itself false if it is not a PR?